tchannel/testing/vcr/cassette.py

```diff
@@ -168,14 +180,20 @@ def _find(self, request, haystack=None):
                 yield interaction
 
     def _load(self):
+        file_hash = None
         try:
             with open(self.path, 'rb') as f:
                 data = f.read()
                 self.existed = True
+                file_hash = sha256(data).hexdigest()
+                if (self.path, file_hash) in self._cache:
+                    self._available = \
+                        deepcopy(self._cache[(self.path, file_hash)])
```

Comment:
Nit: Avoid using `\`. Parens are cleaner.

```python
self._available = deepcopy(
    self._cache[(self.path, file_hash)]
)
```

(Although this shouldn't be a problem if you're using the `self._cache.get(...)` approach instead.)